### DISCUSSION TITLE
Block issue from taking up more quota

### DISCUSSION THREAD

**Discussion by royberris** [getsentry/sentry#35134] [Product Spitballin']
Hi, it would be nice to have a feature where we can stop an issue from taking up more quota. Right now we have the "Delete and Discard". Which will prevent further incoming events on this issue to be blocked, stopping the quota. But this doesn't really fit because now we can't see the issue anymore.

Over the weekend an issue occured 75K times. It was one of those issues that where a side effect, but didn't disrupt the user. Caused by a external library (so no control over the code). But this did eat up our quota, I had to delete and discard it, so it stopped coming in. But now I can't see the issue anymore, when our team has time in the future. We might invest some hours in to trying to avoid the issue, rather then ignoring it. 

So it would be nice for it to still show up in Sentry. But block further incoming events from being registered. Maybe like a toggle, where we can block it for some time. When we actually resolved it, mark it as resolved and see if it comes back in the future. 

Ignoring an issue still captures it.

**Comment by ChristinaDRoberts** [comment]
I am not from Sentry but I do know that there is a way to limit the amount the same error can be sent to sentry by adding to the SDK code. Im still looking for that documentation though as well.

**Reply by cwangsanata** [reply]
Hi @ChristinaDRoberts, did you ever find a way to limit the same errors?
